tests/e2e/actions.go

```diff
@@ -805,6 +942,48 @@ func (tr TestRun) addIbcConnectionGorelayer(
 	tr.waitBlocks(action.chainB, 1, 10*time.Second)
 }
 
+type createIbcClientsAction struct {
+	chainA chainID
+	chainB chainID
+}
+
+func (tr TestRun) createIbcClientsHermes(
```

Comment:
When you run `hermes create connection --a-chain <a> --b-chain <b>` hermes will reate NEW clients for you.

Reply:
Done